src/main/java/frc/robot/Robot.java

```diff
@@ -28,8 +34,32 @@ public void robotInit() {
     // Instantiate our RobotContainer.  This will perform all our button bindings, and put our
     // autonomous chooser on the dashboard.
     m_robotContainer = new RobotContainer();
+		// register subsystems into a list to minimize redundancy if logging
+		// inputs/outputs for all
+		inst = NetworkTableInstance.create();
+
+		// start a NT4 client
+		inst.startClient4("robot network table client");
```

Comment:
this initialization code is only required if you're running the NT server on a non-roborio platform